engine/access/ping/node_info.go

```diff
@@ -0,0 +1,51 @@
+package ping
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+func readJson(jsonFileName string) (map[flow.Identifier]string, error) {
```

Comment:
These are much more specialized json reading functions, lets give them more specific names, maybe `readExtraNodeInfoJSON` or similar. Same with the rest of the functions in this file. (Also, go convention prefers captializing the whole acronym, e.g. JSON instead of Json)